### For tpo_vision: a keypoint on the image

Header header        # Header timestamp should be acquisition time of image
                     # Header frame_id should be optical frame of camera
                     # origin of frame should be optical center of camera
                     # +x should point to the right in the image
                     # +y should point down in the image
                     # +z should point into to plane of the image
                     # If the frame_id here and the frame_id of the CameraInfo
                     # message associated with the image conflict
                     # the behavior is undefined
                     # seq used for image id?

uint16 x_pixel
uint16 y_pixel

uint16 image_width   # width of the source image in pixels
uint16 image_height  # height of the source image in pixels

uint8 label 

float32 confidence # This depend on the method used
bool predicted # True if the keypoint is predicted (no measurement in this frame)
bool depth_assumed_plane # True if depth is unavailable and the point is projected onto the table plane
